lib/ddtrace/transport/http/service.rb

```diff
@@ -0,0 +1,31 @@
+require 'net/http'
+
+module Datadog
+  module Transport
+    module HTTP
+      # An HTTP service which tracer data can be sent to.
+      # Opens and manages connections.
+      class Service
```

Comment:
Should we rename to not confuse with the other "service" we use for traces/spans?

Reply:
What other name would you suggest? Not sure if I have a better suggestion. Maybe `Server`?

Reply:
hmmm... why do you ask such difficult questions...

Agent? Client?

Reply:
There's already a `Client` and `Agent` might be a bit too specific. Basically `Service` is supposed to represent "an addressable process which hosts an API." I thought `Service` was pretty fitting since it's congruent with the idea of "service oriented architecture", but `Server` could work too.

Reply:
In Python our "Client" and "API" are the same class, so we just call it "API".

"Service" could be ok. I really don't have a better suggestion.

Reply:
This `Service` class has been superseded by the `Adapter` class; the adapter manages connectivity to processes/services, and defines the specific delivery mechanism for it (e.g. `Net::HTTP`, Unix sockets, etc.)